packages/docs/versioned_docs/version-2.0.0-alpha-3/protocol/users-and-attesters.md

```diff
@@ -2,18 +2,16 @@
 
 ## Actors in Unirep
 
-### User 👤
+### Users 👤
+**receive reputation** from attesters and are able to **prove received reputation**.
 
-There are users who **receive reputation** and **prove received reputation**.
+* Users sign up by calling `userSignUp` on the Unirep contract with a signup proof.
+* Users' `identityCommitment` is revealed at this time and recorded in the contract to prevent double signup.
+* The identity commitment does not reveal the actual `identity` of the user but allows the user to prove their identity in the circuit.
```

Comment:
i think this last point doesn't make sense. The `identityCommitment` is the user identity as far as the unirep protocol is concerned.

Reply:
How about:
- A user later proves their identity to the protocol by re-creating the `identity commitment` with the secret values known only to the user.

?

Reply:
Yes that works, maybe like "the user _can_ later ..."